css/portal-dashboard/all-responses-popup/popup-header.less

```diff
@@ -0,0 +1,235 @@
+                }
+                &:hover .responseIcon {
+                    fill: @cc-teal;
+                }
+                &:hover .responseTitle {
+                    display: block;
+                }
+            }
+            &.toggleOff .responseIcon {
+                fill: @cc-teal;
+            }
+            &.toggleOff .responseTitle {
+                display: none;
+            }
+            .responseIcon {
+                fill: white;
+                margin: 0 auto 0 ;
+                flex-shrink: 0;
+            }
+            .responseTitle {
+                margin-right: 5px;
+            }
+        }
+        .feedbackToggle {
+            display: flex;
+            order: 2;
+            position: absolute;
+            left: calc(100vw/2);
+            border: 1.5px solid white;
+            border-radius: 0 15px 15px 0 ;
+            justify-content: flex-end;
+            margin-left: 2.5px;
+            padding: 0 5px 0 5px;
+            align-items:center;
+            width: 204px;
+            height: 30px;
+            box-sizing: border-box;
+            background-color: @feedback-green;
+
+            &:hover {
+                background-color: @feedback-green-light2;
+                border-color: @feedback-green;
+            }
+            &:hover .feedbackIcon {
+                fill: @feedback-green;   
+            }
+            &:hover .feedbackTitle{
+                color: black;   
+            }
+            &.toggleOff {
+                width: 35px;
+                background-color: @feedback-green-light4;
+                border-color: @feedback-green-light4;
+                &:hover {
+                    background-color: @feedback-green-light2;
+                    border-color: @feedback-green;
+                    width: 204px;
+                    left: calc(100vw/2);
+                }
+                &:hover .feedbackIcon {
+                    fill: @feedback-green;
+                }
+                &:hover .feedbackTitle {
+                    display: block;
+                }
+            }
+            &.toggleOff .feedbackIcon {
+                fill: @feedback-green;
+                position: relative;
+                left: -2.5px;
+                top: .5px;
+            }
+            &.toggleOff .feedbackTitle {
+                display: none;
+            }
+            .feedbackIcon {
+                fill: white;
+                flex-shrink: 0;
+                margin: 0 auto 0;
+            }
+            .feedbackTitle {
+                margin-left: 5px;
+            }
+        }
+    }
+    .headerRight {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        flex: 1;
+        justify-content: flex-end;
+        margin: 0 20px 0 10px;
+        .closeIcon {
+            width: 27px;
+            height: 27px;
+            background-color: @cc-teal-light6;
+            border-radius: 50%;
```

Comment:
Add `cursor: pointer;` to get the correct cursor state when hovering the button.